scrapers/Meninosonline.yml

```diff
@@ -0,0 +1,38 @@
+name: "MeninosOnline"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - meninosonline.net
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: 
+        selector: //div[@class="col-sm-12"]/h3
+        postProcess:
+          - replace:
+              - regex: .* \(
+                with: ""
+              - regex: \) .*
+                with: ""
```

Comment:
This is not getting a good title
For older scenes the h3 doesnt have any `()` so you get non consistent titles